index.js

```diff
@@ -0,0 +1,79 @@
+var Eris = require('eris')
+
+let pingCount = 0
+const newsChannel = process.env.NEWS_CHANNEL
+
+const bot = new Eris.CommandClient(process.env.TOKEN, {}, {
+    description: "A helpful server bot",
+    owner: "Errolyn",
+    prefix: "!"
+});
+
+bot.on("ready", () => {
+    console.log("Ready!");  
+});
+
+bot.on("error", err => {
+    console.log(err);
+});
+
+bot.registerCommand(
+    "ping", 
+    () => {
+        incPing()
+        return("I have been pinged " + pingCount + " times since I last took a nap.")
+    },
+    {
+        description: "Pong!",
+        fullDescription: "Used to see if the bot is responding."
+    }
+);
+
+bot.registerCommand(
+    "news", 
+    (msg) => {
+        const userName = msg.author.username;
+        const messageChannelName = msg.channel.name;
+        const contentForNewsChannel = stripContent(msg.content);
+
+
+        let attachments = getAttachments(msg.attachments);
+        let content = `${userName} posted in ${messageChannelName}: \n${contentForNewsChannel} \n${attachments}`;
+
+        createMessage(newsChannel, content, attachments);
+    }, 
+    {
+        description: "News feed",
+        fullDescription: "Use this command to add content to #news-feed."
+    }
+)
+
+bot.connect();
+
+// helper functions
+
+function incPing(){
+    pingCount++
+}
+
+function stripContent(messageContent){
+    const stringParts = messageContent.split(' ');
+    stringParts.shift();
+
+    const userPost = stringParts.join(' ');
+    return(userPost);
+}
+
+function createMessage(channel, content){
+    bot.createMessage(channel,{
+        content: content,
+    })
+}
+
+function getAttachments(attachments){
+    const attachmentString = []
+    if(attachments !== undefined){
+        attachments.map( (attachment) => { attachmentString.push(`\n ${attachment.url}`) })
+    }
+    return(attachmentString.join(''))
+}
```

Comment:
The process I see here is

1. Accept a list an array of attachments, or nothing
2. Extract the URL from each attachment
3. Join them all with newlines (and spaces?)

I'd write this as

```suggestion
function getAttachments(attachments = []) {
	return attachments.map(attachment => attachment.url).join('\n')
}
```

Giving `attachments` a default value means JS will do the `if (attachments !== undefined)` check for us. Then moving the new line out of the `map` and into the `join` lets each do it's job more clearly.

In other places I've suggested removing single line functions, but in this case I think there is enough complexity it is worth keeping it as a function. I would probably actually format it as this, but I don't know the automatic semi-colon insertion rules well enough to be sure it would work.

```
function getAttachments(attachments = []) {
	return attachments.map(attachment => attachment.url)
		.join('\n');
}
```

I'd also recommend renaming this as `formatAttachments`, since it is doing more than just getting them from the list.